### PR TITLE
OTHER Update Readme

### PR DESCRIPTION
**Pull request by elishkaflint**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,81 @@
-# AceBook
+# Acebook
 
-Fork this repository to `acebook-teamname` and customize
-the below**
+## Summary
 
-[You can find the engineering project outline here.](https://github.com/makersacademy/course/tree/master/engineering_projects/rails)
+Week 8 and 9 Makers Academy engineering project to develop an Facebook clone, called Acebook. <br>
```
Comment (neobay991) [inline README.md]: typo -
should be '... a Facebook clone...' 
and not '... an Facebook clone... '

File: README.md [hunk 2]
```diff
@@ -1,16 +1,81 @@
...
+So that I can curate my content,
+I want to be able to update and delete my posts
+```
+___
+
+## Technical
+
+### Tech/Frameworks Used
+
+Ruby on Rails, Sinatra, CSS, RSpec, Capybara
```
Comment (neobay991) [inline README.md]: I don't think Sinatra is part of the Tech Stack, the server running seems to be Puma (when you run 'bin/rails server' in the command line). Also, there's no Sinatara gem in the Gemfile

**Review by neobay991 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (elishkaflint) [reply README.md]: Thanks - updated

File: README.md (hunk 1 above)
Comment (elishkaflint) [reply README.md]: Thanks - updated

File: .idea/acebook-rails-keeping-it-rails.iml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="PROVIDED" name="diff-lcs (v1.3, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-core (v3.7.1, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-expectations (v3.7.0, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-mocks (v3.7.0, RVM: ruby-2.5.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="rspec-support (v3.7.1, RVM: ruby-2.5.1) [gem]" level="application" />
+  </component>
+</module>
```
Comment (neobay991) [inline .idea/acebook-rails-keeping-it-rails.iml]: This files shouldn't be in this commit?

File: .idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="ES6" />
+  </component>
+</project>
```
Comment (neobay991) [inline .idea/misc.xml]: This files shouldn't be in this commit?

File: .idea/modules.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/acebook-rails-keeping-it-rails.iml" filepath="$PROJECT_DIR$/.idea/acebook-rails-keeping-it-rails.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (neobay991) [inline .idea/modules.xml]: This files shouldn't be in this commit?

File: .idea/vcs.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (neobay991) [inline .idea/vcs.xml]: This files shouldn't be in this commit?

File: spec/features/user_can_sign_up_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+      visit "/"
+      fill_in :first_name, with: "Elishka"
+      fill_in :last_name, with: "Flint"
+      fill_in :email_address, with: "elishka@keepingitrails.com"
+      fill_in :password, with: "pa55w0rd"
+      click_button "Sign Up"
+      expect(page).to have_content("Welcome Elishka!")
+    end
+
+end
```
Comment (neobay991) [inline spec/features/user_can_sign_up_spec.rb]: This files shouldn't be in this commit?

**Review by neobay991 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**